go-controller/pkg/ovn/address_set/fake_address_set.go

```diff
@@ -155,6 +155,24 @@ func (f *FakeAddressSetFactory) removeAddressSet(name string) {
 	delete(f.sets, name)
 }
 
+// expectAddressSetHasIPs ensures the named address set has the given set of IPs
+func (f *FakeAddressSetFactory) expectAddressSetHasIPs(g gomega.Gomega, dbIDs *libovsdbops.DbObjectIDs, ips []string) {
+	as := f.getAddressSet(dbIDs)
+	gomega.Expect(as).ToNot(gomega.BeNil(), fmt.Sprintf("expected address set %s to exist", dbIDs.String()))
```

Comment:
I am not sure I understand your suggestion. The problem here is that IP addresses in FakeAddressSetFactory is undetermined, it may have both lrpIP and NodeMgmtPortIP, or just lrpIP